src/main/scala/com/stratio/receiver/RabbitMQInputDStream.scala

```diff
@@ -98,13 +98,14 @@ class RabbitMQReceiver(params: Map[String, String], storageLevel: StorageLevel)
       log.info("RabbitMQ Input waiting for messages")
       val consumer: QueueingConsumer = new QueueingConsumer(channel)
       log.info("start consuming data")
-      channel.basicConsume(queueName, true, consumer)
+      channel.basicConsume(queueName, false, consumer)
```

Comment:
I could, but I fail to see the point. This value specifies if you would like to auto acknowledge the messages. This means that as soon as the delivery is complete i.e. at line 101 the message is ack'ed and will be removed from the rabbit queue so if line 107 fails for some reason the message is gone and cannot be recovered. 
If autoack is enabled then line 108 is redundant also.

https://www.rabbitmq.com/releases/rabbitmq-java-client/v2.7.1/rabbitmq-java-client-javadoc-2.7.1/com/rabbitmq/client/Channel.html#basicConsume(java.lang.String, boolean, com.rabbitmq.client.Consumer)